frontend/src/components/UseCategoryDropdown.vue

```diff
@@ -1,39 +1,52 @@
 <template>
-  <select @change="$emit('selected', $event.target.value)" class="UseCategoryDropdown">
-    <option value="">Everything</option>
-    <option :value="category" v-for="category in categories">
-      {{ category }} Uses
-    </option>
-  </select>
+<b-dropdown :text="displayCategory(category)" class="UseCategoryDropdown" right>
+  <b-dropdown-item-button :key="category" class="category-item" @click="onCategorySelected('')">Everything</b-dropdown-item-button>
```

Comment:
Remove `:key="category"` from this one. Only need in conjunction with the `v-for` below.